fels/fels.py

```diff
@@ -4,56 +4,51 @@
 import datetime
 import os
 import json
-import pkg_resources
-from fels.utils import *
-from fels.landsat import *
-from fels.sentinel2 import *
-import geopandas
-import json
-import shapely as shp
-
-
-def convert_wkt_to_scene(sat, geometry, include_overlap):
```

Comment:
Why are you removing this function?

Reply:
I think I just moved it. Give me a day or two, I can clean up this PR so it is more reviewable. Thanks for merging the other one!